api/commands.go

```diff
@@ -182,6 +182,9 @@ type NewContainer struct {
 	Fsmap            []Fsmap          `json:"fsmap"`
 	Process          Process          `json:"process"`
 	SystemMountsInfo SystemMountsInfo `json:"systemMountsInfo"`
+
+	// SCSI address in the format SCSI-Id:LUN
+	SCSIAddr string `json:"scsiAddr,omitempty"`
```

Comment:
I am not sure this is the right place for this field. This is very specific to a type of rootfs. Maybe we should define a structure Rootfs so that we can actually include this type of information. WDYT ?

Reply:
Yes, this is specific to rootfs, but we have other fields here like Rootfs, FsMap and FsType that are related to rootfs that Iwould be candidates for a combined structure. I would like to do that in a separate PR, as this involves a lot of components.

Reply:
Fair enough, but please open an issue to track this down ;)

Reply:
Sure will do.

Reply:
https://github.com/clearcontainers/agent/issues/210

Reply:
thx